sopel/plugins/handlers.py

```diff
@@ -310,7 +382,13 @@ class PyFilePlugin(PyModulePlugin):
     In this example, the plugin ``custom`` is loaded from its filename despite
     not being in the Python path.
     """
+
     PLUGIN_TYPE = 'python-file'
+    """Plugin's type.
+
+    Metadata for the plugin, this should be considered to be a constant and
+    should not be modified at runtime.
+    """
```

Comment:
```suggestion
    """The plugin's type.

    Metadata for the plugin; this should be considered to be a constant and
    should not be modified at runtime.
    """
```

Ditto note for `PLUGIN_TYPE` constant in `PyModulePlugin`.